### ISSUE TITLE
"No loader found for resource: res://" when script defines a class

### ISSUE THREAD

**Issue by hilfazer** [godotengine/godot#22377]
**Godot version:**

8b2eeaa724829445b1f8b65855a816003df4f2ba

**OS/device including version:**

Windows 7 SP1

**Issue description:**

Message "No loader found for resource: res://" gets spammed to Output when selecting a node with a script that defines a class.

**Steps to reproduce:**
Run reproduction project and select 'Node' in Scene window.

**Minimal reproduction project:**

https://github.com/hilfazer/Projects/tree/EngineIssues/Godot/NoLoaderResourceClass

**Comment by akien-mga** [comment]
See also #22376.

**Comment by DualMatrix** [comment]
Preventing the spam was easy but in this case there was some real oddness, for some reason the Class C shows up under constants in the remote tree as an empty resource. I have no idea what causes this, I'll have to dig deeper.

**Comment by Lucrecious** [comment]
I have the same issue as OP, but this is with Godot Version 3.1 beta 5. I get spammed with the error as soon as I click "Add Node" and while I'm typing in the "Add Node" search.

I used lldb with VSCode to figure out where the `No loader found for resource: res://` was coming from (it'd be cool to get some kind of hint as to "who" is trying to load this resource).

I found out that it's somehow coming from the NativeScript classes I have. As in, there's one `No loader found ...`  for each `.gdn` I'm using. All `.gdn`s have a non-empty class name, script name, valid library path, valid resource path to itself and a resource name under the inspector.

Here's an example of one of my `.gdn`s:
```
[gd_resource type="NativeScript" load_steps=2 format=2]

[ext_resource path="res://bin/lutlib.gdnlib" type="GDNativeLibrary" id=1]

[resource]

resource_name = "CharacterAStar"
class_name = "CharacterAStar"
library = ExtResource( 1 )
script_class_name = "CharacterAStar"
```

Here's how the project.godot looks like for defining this global class name:
```
{
"base": "Reference",
"class": "CharacterAStar",
"language": "NativeScript",
"path": "res://src/lut/native/scripts/character_astar.gdns"
}
```

The spamming seems to really slow down the "Add Node" window. Any thoughts? @DualMatrix @akien-mga
